src/components/LoanCollections/KivaClassicLoanCarousel.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+	<div class="tw-pt-4">
+		<transition name="kvfade">
+			<div
+				v-if="isLoading"
+				class="spinner"
+			>
+				<kv-loading-spinner />
+			</div>
+		</transition>
+
+		<kv-carousel
+			v-if="augmentedLoanIds.length > 0 && isVisible"
+			class="tw-w-full tw-overflow-visible md:tw-overflow-hidden"
+			:embla-options="{
+				loop: false,
+			}"
+			:multiple-slides-visible="true"
+			slides-to-scroll="visible"
+			:slide-max-width="singleSlideWidth"
+			@interact-carousel="onInteractCarousel"
+		>
+			<template v-for="(loanId, index) in augmentedLoanIds" #[`slide${index}`]>
+				<!-- Show View more Card -->
+				<router-link
+					v-if="loanId === 3"
+					:key="`view-more-card-${loanId}`"
+					class="tw-flex tw-items-center tw-h-full tw-w-full
+						hover:tw-bg-action-highlight hover:tw-text-primary-inverse tw-rounded"
+					:to="cleanUrl"
+					v-kv-track-event="[
+						'Homepage',
+						'click-carousel-view-all-category-loans',
+						`${viewAllLoansCategoryTitle}`]"
+				>
+					<div class="tw-w-full tw-text-center">
+						<h3>{{ viewAllLoansCategoryTitle }}</h3>
+					</div>
+				</router-link>
+
+				<!-- show loan card -->
+				<!-- TODO Re-implement card position analytics -->
+				<kiva-classic-basic-loan-card
+					v-else
+					:item-index="index"
+					:key="`loan-${loanId}`"
+					:loan-id="loanId"
+				/>
+			</template>
+		</kv-carousel>
+	</div>
+</template>
+
+<script>
+import KivaClassicBasicLoanCard from '@/components/LoanCards/KivaClassicBasicLoanCard';
+import KvLoadingSpinner from '@/components/Kv/KvLoadingSpinner';
```

Comment:
Can we use KvLoadingSpinner from kv-components? https://608b4cf87f686c00213841b1-pkurhylpcj.chromatic.com/?path=/docs/kvloadingspinner--single-loader

Reply:
Will do before launch!